sensu-plugins-ftp.gemspec

```diff
@@ -29,7 +29,9 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsFtp::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin',                  '~> 1.2'
+  s.add_runtime_dependency 'net',                           '~> 0.3.3'
```

Comment:
The net gem "Retrieves information for Twitter users". Did you mean something else?